squad/frontend/templates/squad/_builds_table.html

```diff
@@ -7,6 +7,26 @@
 <div class="row row-bordered build">
     <div class="col-md-2 col-sm-2">
         {% if not build.status.finished %}<i title="Build/testing in progress" class="fa fa-cog fa-spin"></i>{% endif %}
+        <i class="fa fa-exclamation-triangle text-danger" ng-show="regressions[{{build.id}}] != null" id="popperRegressions{{ build.id }}"></i>
+        <i class="fa fa-check-circle text-success popper" ng-show="fixes[{{build.id}}] != null" id="popperFixes{{ build.id }}" ></i>
+        <div class="hidden">
+            <div id="regressions{{ build.id }}">
+                <div ng-repeat="(regression_key, regression_value) in regressions[{{ build.id }}]">
+                    <h3><span ng-bind="regression_key"></span></h3>
+                    <ul>
+                        <li ng-repeat="test in regression_value"><span ng-bind="test"></span></li>
+                    </ul>
+                </div>
+            </div>
+            <div id="fixes{{ build.id }}">
+                <div ng-repeat="(fix_key, fix_value) in fixes[{{ build.id }}]">
+                    <h3><span ng-bind="fix_key"></span></h3>
+                    <ul>
+                        <li ng-repeat="test in fix_value"><span ng-bind="test"></span></li>
+                    </ul>
+                </div>
+            </div>
+        </div>
```

Comment:
you could just generate the data server side here, and keep the JS code just to show/hide the popover.

Reply:
This isn't so obvious. The field value is serialized. Converting to object is done with JS. If this is done on server side at least templatetag is required.